### ISSUE TITLE
Is a Range Slider component coming?

### ISSUE THREAD

**Issue by delueg** [marcosmoura/vue-material#115]
Hello,

i want to ask if you  planning to implement a <input type="range" /> component for vue-material.

All Best

**Comment by marcosmoura** [comment]
Yep. Will be developed. But unfortunately there's some other components or improvements to be made. So this will be scheduled for 0.6.0 or 0.7.0. Thank you.

**Comment by copitz** [comment]
I've got a vue material slider component ready using [noUiSlider](https://refreshless.com/nouislider/) - let me know if you want me to contribute it

**Comment by Xeoncross** [comment]
Not supporting range-slider is a big deal breaker for me. Here is a basic CSS-only slider if anyone is interested in how it might work: https://codepen.io/Xeoncross/pen/dRJmpj

**Comment by copitz** [comment]
@Xeoncross you can use my component for this if you want: https://github.com/netresearch/continnous/tree/master/src/md/mdSlider

I already had a short discussion with @marcosmoura about that and he was interested in this component as well - however I didn't have time yet to create a pull request for that - maybe you have? ;)

![image](https://user-images.githubusercontent.com/882663/27725707-d8582c08-5d77-11e7-8108-642db1400a47.png)

**Comment by Samuell1** [comment]
Sooon will be added! 😃 

![obrazok](https://user-images.githubusercontent.com/3110002/35477211-d9717002-03be-11e8-9a38-2efeaa5b64a6.png)

**Comment by mcicolella** [comment]
Is there already a deadline for this?

**Comment by Samuell1** [comment]
@mcicolella check PR, its finished on other browsers but doesnt work on IE yet

**Comment by mcicolella** [comment]
Any update about this?

**Comment by Samuell1** [comment]
@mcicolella i had working slider but it was based on native input and i run in some problems with it (cross browser support, advanced styles, buble icon, ticks, ...), that means we need to make it separate from that native input if we want to use some advanced things for it.

**Comment by jjtbsomhorst** [comment]
Any update? Is it already usable?

**Comment by rainabba** [comment]
Also curious as I'm hunting for a viable slider component.

**Comment by jimmywarting** [comment]
what was the problem with ie and native input type range?

according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) ie10 have support for range input?

or is it problem with styling?

Can't wait until edge becomes chromium based!

**Comment by Samuell1** [comment]
@jimmywarting mostly issue was you need to maintain code for every browser separately because every browser has other definitions and some browser are missing features of styling ticks, bubles etc thats why best approach is to create own component with divs and bind value to slider input

**Comment by jjtbsomhorst** [comment]
Isn't the idea of this project that such things are taking care of and you don't need to invent the wheel yourself?

**Comment by Samuell1** [comment]
@jjtbsomhorst yes, but iam talking about slider component that it needs to be created with divs etc to fix many issues between browsers and support other features. I think i wrote it badly before and it sounds like "you" need to make it in your project :D
